src/jit/emit.h

```diff
@@ -923,11 +925,12 @@ class emitter
 
             struct
             {
-                regNumber _idReg3 : REGNUM_BITS;
-                regNumber _idReg4 : REGNUM_BITS;
 #ifdef _TARGET_ARM64_
-                unsigned _idReg3Scaled : 1; // Reg3 is scaled by idOpSize bits
+                emitLclVarAddr iiaLclVar;
```

Comment:
@briansull BTW.  There are 19 bits left here.  The ldp/stp second gcAttributes might be able to move into here to recover the lost const size.